scripts/apps/authoring/views/change-image.html

```diff
@@ -1,156 +1,280 @@
+                                    <option translate value="single-usage">Single Usage</option>
+                                    <option translate value="time-restricted">Time Restricted</option>
+                                    <option translate value="indefinite-usage">Indefinite Usage</option>
+                                </select>
+                            </div>
+                        </div>
+                        <div class="form__row" ng-if="validator.copyrightnotice">
+                            <div class="sd-line-input sd-line-input--dark-ui sd-line-input--boxed" ng-class="{'sd-line-input--required':validator.copyrightnotice.required}">
+                                <label class="sd-line-input__label" translate>Copyright notice</label>
+                                <input class="sd-line-input__input" type="text" ng-model="data.metadata.copyrightnotice" ng-change="data.isDirty = true" ng-disabled="!data.editable">
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="sd-slide-in-panel__content" ng-if="nav === 'image-edit'">
+                    <div class="sd-slide-in-panel__content-block sd-slide-in-panel__content-block--first">
+
+                        <div class="modal__sliding-toolbar" ng-if="isAoISelectionModeEnabled && !loaderForAoI">
+                            <div class="sliding-toolbar__inner"></div>
+                            <a class="btn" ng-click="showAreaOfInterestView(false)" translate>Cancel</a>
+                            <a class="btn btn--primary" ng-click="saveAreaOfInterest(areaOfInterestData)" ng-disabled="!isAoIDirty" translate>Confirm crop</a>
+                        </div>
+                        
+                        <div class="modal__sliding-toolbar" ng-if="controls.isDirty">
+                            <div class="sliding-toolbar__inner"></div>
+                            <a class="btn" ng-click="cancelImageChanges()" translate>Cancel</a>
+                            <a class="btn btn--primary" ng-click="applyImageChanges()" translate>Apply</a>
+                        </div>
+
+                        <h3 class="sd-slide-in-panel__heading sd-slide-in-panel__heading--marg-b16" translate>Change image settings</h3>
+                        <div class="form__row">
+                            <button class="btn btn--ui-dark btn--hollow btn--icon-only btn--large" ng-click="showAreaOfInterestView()" ng-class="{'btn--active' : isAoISelectionModeEnabled}" sd-tooltip="Crop" ng-disabled="controls.isDirty"><i class="icon-crop icon--white"></i></button>
+                            <button class="btn btn--ui-dark btn--hollow btn--icon-only btn--large" ng-click="controls.rotate = controls.rotate - 90; controls.isDirty = true" sd-tooltip="Rotate left" ng-disabled="isAoISelectionModeEnabled"><i class="icon-rotate-left icon--white"></i></button>
```

Comment:
imo would be goot to have functions for those rotations, it's not very intuitive to define it in html. makes it hard to test it